css-canvas-medical/index.html

```diff
@@ -0,0 +1,118 @@
+          <img src="images/logo-blue.svg" alt="logo-blue.svg">
+        </div>
+        <div class="column-half">
+          <div class="row" id="header-link">
+            <div><a href="">Product</a></div>
+            <div><a href="">Implementation</a></div>
+            <div><a href="">About Us</a></div>
+            <div><a href="link-a">Blog</a></div>
+            <div><a href="" id="button-header">Contact Us</a></div>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="row">
+        <div class="column-full" id="row2">
+          <div class="background">
+            <div class="overlay"></div>
+            <h2>The EMR Reimagined for <br> Primary Care</h2>
+            <p>Can't remember the last time you talked with a patient without focusing <br>
+              on the EMR? Canvas lets you rediscover joy in the work you do. Capture <br>
+              meaningful data, identify gaps in care, and move seamlessly between <br>
+              tasks within the platform. You stay present with patients while your <br>
+              whole care team stays up-to-speed. Be a better doctor with Canvas, <br>
+              designed for primary care providers with the help of primary care <br>
+              providers.</p>
+            <button>Get Started</button>
+          </div>
+        </div>
+      </div>
+      <div class="row" id="row3">
+        <div class="column-full" id="row3-text">
+          <h3>Clinicians Prefer Canvas</h3>
+          <p>Physician-owned primary care practices across the country have <br> already made Canvas the fabric of their
+              care
+          </p>
+        </div>
+      </div>
+      <div class="row" id="row4">
+        <div class="column-sixth">
+          <div class="column-one-sixth">
+            <p>Cascade Family Practice, Portland OR</p>
+          </div>
+          <div class="column-one-sixth">
+            <p>Ward Medical Clinic, Bonifay FL</p>
+          </div>
+          <div class="column-one-sixth">
+            <p>Temescal Creek Medicine, Oakland CA</p>
+          </div>
+          <div class="column-one-sixth">
+            <p>Caribou Medical, Towanda PA</p>
+          </div>
+          <div class="column-one-sixth">
+            <p>San Juan Healthcare, Friday Harbor WA</p>
+          </div>
+          <div class="column-one-sixth">
+            <p>Bay Tree Family Care, Red Bay AL</p>
+          </div>
+        </div>
+      </div>
+      <div class="row" id="row5">
+        <div class="column-half" id="row5-text">
+          <h3>Ready to learn more?</h3>
+          <p>Schedule a private meeting now</p>
+        </div>
+        <div class="column-half" id="row5-button">
+          <button class="button">Get Started</button>
+        </div>
+      </div>
+      <div class="row" id="row6">
+        <div class="column-half">
+          <img src="images/logo-white.png" alt="logo-white.png" id="row6-img">
+          <address id="row6-address">
```

Comment:
Do not create numbered ids or class names, it is lazy. Create names that accurately describe what they represent.